Doc/Tutorial/chapter_entrez.tex

```diff
@@ -48,6 +48,15 @@ \section{Entrez Guidelines}
 \end{verbatim}
 The tool parameter will default to Biopython.
 \item For large queries, the NCBI also recommend using their session history feature (the WebEnv session cookie string, see Section~\ref{sec:entrez-webenv}).  This is only slightly more complicated.
+\item After May 1, 2018, NCBI will limit access to the E-utilities unless one have an API key (more info at \url{https://ncbiinsights.ncbi.nlm.nih.gov/2017/11/02/new-api-keys-for-the-e-utilities/}).
+Without a valid key, a maximum of 3 queries per second is allowed. With a valid key, this limit can be increased to 10 queries per second. The \verb|Bio.Entrez| module handles the limit automatically when the right parameter is specified in any E-utility (include {\tt api_key="MyAPIkey"} in the argument list):
+%doctest
+\begin{verbatim}
+>>> from Bio import Entrez
+>>> Entrez.email = "A.N.Other@example.com"
+>>> handle = Entrez.efetch(db='protein', id='15718680,157427902,119703751',
+                               retmode='xml', api_key="MyAPIkey")
+\end{verbatim}
```

Comment:
Without trying the code, I am assuming you _can_ supply ``api_key=...`` here as an argument, but it would be neater to do it once like the email address in this example.